### PR TITLE
HW7 DONE

### PR DESCRIPTION
**Pull request by skoubye**

### CODE REVIEW

File: hw7/_submissions/skoubye-steven/camel-case/src/camel-case.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { convertToCamelCase } from './camel-case'
+
+describe('When converting to camel case', () => {
+  it('should convert two words', () => {
+    const result = convertToCamelCase('hello case')
+    const expected = 'helloCase'
+
+    expect(result).toEqual(expected)
+  })
+  // TODO: Write more tests
```
Comment (patrik-drean) [inline hw7/_submissions/skoubye-steven/camel-case/src/camel-case.test.js]: Another test might have been nice, like if only one word is passed or an empty string.
